docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -338,16 +369,18 @@ The following information is returned.
 
 * `YIELD` specifies what values or results you might want to return from the query.
 * `$^` represents the source vertex.
-* `$$` indicates the target vertex.
+* `$$` represents the target vertex.
+* `\`  represents a line break.
 
 **Example 3**. Find the team which is served by the player who is followed by `100`.
 
 There are two ways to get the same result. First, we can use a `pipe` to retrieve the team. Then we use a `temporary variable` to retrieve the same team.
 
 Enter the following statement with a `pipe`:
 
-```
-GO FROM 100 OVER follow YIELD follow._dst AS id | GO FROM $-.id OVER serve YIELD $$.team.name AS Team;
+```ngql
+GO FROM 100 OVER follow YIELD follow._dst AS id | \
+GO FROM $-.id OVER serve YIELD $$.team.name AS Team;
```

Comment:
ditto

Reply:
I just wrapped the line after the backslash. Could not figure out why two spaces.